recipe/meta.yaml

```diff
@@ -5,18 +5,26 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/p/pythran/pythran-{{ version }}.tar.gz
-  sha256: 9dac8e1d50f33d4676003e350b1f0c878ce113e6f907920e92dc103352cac5bf
+  url: https://github.com/serge-sans-paille/pythran/archive/refs/tags/{{ version }}.tar.gz
+  sha256: c4853430e99dc382b8ed11e6eed3a196a571b2f311db4be4cf7b28d6f6866c54
```

Comment:
This is not strictly speaking necessary, but allows easier patchability (there were some issues using PyPI sources). It's also more transparent, IMO.